WLC/web_endpoint.py

```diff
@@ -81,8 +82,31 @@ def api_resubmit_code():
 
 
 def _get_ar_coordinates(pic, errors):
+    # calculate mins, maxs
+    min_x = min_y = sys.maxsize
+    max_x = max_y = 0
+
+    i = 1
+    line = pic.get_line_coordinates(i)
+    while line:
+        if line['x'] < min_x:
+            min_x = line['x']
```

Comment:
Doing `min_x = min(min_x, line['x'])` might be nicer because you dont need all the ifs